docs/dev/api-usage.rst

```diff
@@ -0,0 +1,93 @@
+.. role:: py(code)
+   :language: python
+
+API Usage Examples
+==================
+
+.. note::
+
+   These examples are written in Python version **3.6.3** and use `requests
+   <http://docs.python-requests.org/en/master/>`_ version **2.18.4**.
+
+Prerequisites
+#############
+
+All the examples assume you have a development version of the ListenBrainz
+server set up on :code:`localhost`. Remember to set :code:`DEBUG` to :py:`True`
+in the config. When in production, you can replace :code:`localhost` with
+:code:`api.listenbrainz.org` to use the real API. In order to use either
+one, you'll need a token. You can find it under :code:`ROOT/profile/`
+when signed in, with :code:`ROOT` being either :code:`localhost` or
+:code:`api.listenbrainz.org`.
```

Comment:
We don't want to encourage users to use `api.listenbrainz.org` for anything other than sending API requests. We should change this to `listenbrainz.org`.

Reply:
I don't quite get what you mean here, apologies. Do you mean for the token? Or in general? Because in production, AFAIK, you have to use `api.listenbrainz.org`, right?

Reply:
In general, you should get your production auth token from `listenbrainz.org` instead of `api.listenbrainz.org`. `listenbrainz.org` is the user facing site while `api.listenbrainz.org` is the place which handles the API calls.